customfieldnames.civix.php

```diff
@@ -346,3 +346,166 @@ function _customfieldnames_civix_civicrm_alterSettingsFolders(&$metaDataFolders
     $metaDataFolders[] = $settingsDir;
   }
 }
+
```

Comment:
all hook implementations (including helper functions) should be in customfieldnames.php. 
the customfieldnames.civix.php file is a generated file (by the civix command line tool) that may be overwritten/updated by civix at times. it should not be directly modified.
in this case -- you can simply move the functions you added here to customfieldnames.php and it should work fine.

Reply:
I have moved the functions to customfieldnames.php. Kindly check